src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -78,6 +78,7 @@
         "symfony/messenger": "<4.3",
         "symfony/mime": "<4.4",
         "symfony/property-info": "<3.4",
+        "symfony/security-bundle": "<4.4",
```

Comment:
do we actually need such conflict rule ?

Reply:
If you have a Symfony 4.3 application with an event subscriber registered to, let's say, `SwitchUserEvent::class`, that application would break if you upgraded FrameworkBundle to 4.4 without upgrading SecurityBundle as well.